internal/interpreters/python_requires.go

```diff
@@ -162,9 +162,18 @@ func adaptPythonRequires(raw string) (string, error) {
 		return "==" + constraint, nil
 	}
 
-	if strings.Count(constraint, ".") == 1 {
-		// e.g. 3.8, what the user want is 3.8.*
-		constraint = constraint + ".0"
+	return adaptToCompatibleConstraint(constraint), nil
+}
+
+func adaptToCompatibleConstraint(constraint string) string {
```

Comment:
It might be good to have this shared method as part of a new file (e.g: `interpreters.go`) within the same package, just so to be a bit more clear that it is not hardly associated with python

Reply:
I wanted to keep it close to the "requires" code,
It probably deserves a better packaging story for `r_require` and `python_require` as they are currently mixed in interpreters, but in reality are unrelated to a specific interpreter at all.

For the moment I'd keep it here close to the code that uses it, but I'm happy to revise it in a future PR if there are suggestions on how to refactor the package related to requirements parsing for R and Python